lib/matplotlib/testing/__init__.py

```diff
@@ -105,6 +105,16 @@ def subprocess_run_for_testing(command, env=None, timeout=60, stdout=None,
             import pytest
             pytest.xfail("Fork failure")
         raise
+    except subprocess.CalledProcessError as e:
+        if e.stdout:
+            _log.info(f"Subprocess output:\n{e.stdout}")
```

Comment:
```suggestion
            _log.error(f"Subprocess output:\n{e.stdout}")
```
Even though it is stdout, if the process raised an error, everything about it is error level.